x-pack/plugin/spatial/build.gradle

```diff
@@ -12,6 +12,7 @@ esplugin {
 
 dependencies {
   compileOnly project(path: ':modules:legacy-geo')
+  compileOnly project(path: ':modules:aggregations')
```

Comment:
I think this is only used in test, testImplementation would be better?

Reply:
Done